src/Command/Yaml/DiffCommand.php

```diff
@@ -177,35 +170,52 @@ protected function execute(InputInterface $input, OutputInterface $output)
      */
     protected function interact(InputInterface $input, OutputInterface $output)
     {
-        $validator_filename = function ($value) {
+        $io = new DrupalStyle($input, $output);
+
+        $validator_filename = function ($value) use ($io) {
             if (!strlen(trim($value))) {
-                throw new \Exception(' You must provide a valid file path.');
+                // @todo: translate
+                $io->error(' You must provide a valid file path.');
+
+                return false;
             }
 
             return $value;
         };
 
-        $output = new DrupalStyle($input, $output);
-
         // --yaml-left option
         $yaml_left = $input->getArgument('yaml-left');
         if (!$yaml_left) {
-            $yaml_left = $output->ask(
-                $this->trans('commands.yaml.diff.questions.yaml-left'),
-                null,
-                $validator_filename
-            );
+            while (true) {
```

Comment:
Hi @mnico 

I don't understand why we need a while(true) if the ask fail the cycle never ends